### PR TITLE
Extend IBC register account API

### PR DESCRIPTION
**Pull request by Kayanski**
Requires https://github.com/AbstractSDK/abstract/pull/137 to be merged before merging

This PR aims at allowing users to configure their account when registering them across IBC in 1 transaction.

### Checklist

- [x] CI is green.
- [x] Changelog updated.

### CODE REVIEW

File: framework/contracts/account/proxy/src/commands.rs [hunk 1]
```diff
@@ -417,6 +417,9 @@ mod test {
             let msg = ExecuteMsg::IbcAction {
                 msgs: vec![abstract_core::ibc_client::ExecuteMsg::Register {
                     host_chain: "juno".into(),
+                    base_asset: None,
+                    namespace: None,
+                    install_modules: vec![],
                 }],
```
Comment (Buckram123) [inline framework/contracts/account/proxy/src/commands.rs:423]: Let's add tests for those fields as well! Just one test with all set to make sure it don't panic anywhere unexpectedly

File: integrations/kujira-adapter/Cargo.toml [hunk 2]
```diff
@@ -25,5 +25,5 @@ cw-utils = { workspace = true, optional = true }
 
 [dev-dependencies]
 abstract-dex-standard = { workspace = true, features = ["testing"] }
-cw-orch = { workspace = true }
+cw-orch = { workspace = true, features = ["daemon"] }
```
Comment (Buckram123) [inline integrations/kujira-adapter/Cargo.toml:28]: Is this feature used anywhere?

**Review by Buckram123 — CHANGES_REQUESTED** [review]
Nice update! Would like to see one small test of it

File: integrations/kujira-adapter/Cargo.toml (hunk 2 above)
Comment (CyberHoward) [reply integrations/kujira-adapter/Cargo.toml:28]: nope, removed

File: framework/contracts/native/ibc-host/tests/integration.rs [hunk 3]
```diff
@@ -87,6 +105,88 @@ fn account_creation() -> anyhow::Result<()> {
...
+#[test]
+fn account_creation_full() -> anyhow::Result<()> {
+    let sender = Addr::unchecked("sender");
+    let chain = Mock::new(&sender);
+
+    let admin = Addr::unchecked("admin");
+    let mut admin_chain = chain.clone();
+    admin_chain.set_sender(admin.clone());
+
+    let admin_abstr = Abstract::deploy_on(admin_chain.clone(), admin.to_string())?;
```
Comment (CyberHoward) [inline framework/contracts/native/ibc-host/tests/integration.rs:117]: `admin_abstr` has confused me and @AndrewPochapsky. Can we call it `home_abstr` and the chain `home_chain`?

Comment (Buckram123) [reply framework/contracts/native/ibc-host/tests/integration.rs]: Agree on this, updated!

File: app-template/Cargo.toml
Comment (CyberHoward) [inline app-template/Cargo.toml:1]: These changes should not be in here

File: framework/Cargo.toml [hunk 4]
```diff
@@ -106,3 +106,6 @@ debug-assertions = false
 codegen-units = 1
 panic = 'abort'
 incremental = false
+
+[patch.crates-io]
```
Comment (CyberHoward) [inline framework/Cargo.toml:110]: This needs to be removed

**Pull request merged**